apps/studio/src/lib/db/clients/bigquery.js

```diff
@@ -276,6 +422,40 @@ export async function selectTop(client, db, table, offset, limit, orderBy, filte
   return result
 }
 
+export async function selectTopStream(conn, db, table, orderBy, filters, chunkSize) {
+  const bqTable = db + "." + table
+  const qs = buildSelectTopQuery(bqTable, null, null, orderBy, filters);
+  const columns = await listTableColumns(conn, db, table);
+  console.log('PARAMS: ', {
```

Comment:
Do we still want this popping up in the console? I know the client files are peppered with them but 🤷

Reply:
Grrrr I thought I got em all

Reply:
log.debug plz